libraries/scripting/dependencies/src/kotlin/script/experimental/dependencies/EqualityExternalDependenciesResolverOptionsParser.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
+ * Use of this source code is governed by the Apache 2.0 license that can be found in the license/LICENSE.txt file.
+ */
+
+package kotlin.script.experimental.dependencies
+
+private val nameRegex = Regex("^[^\\S\\r\\n]*([a-zA-Z][a-zA-Z0-9-_]*)\\b")
+private val valueRegex = Regex("^[^\\S\\r\\n]*([a-zA-Z0-9-_]+)\\b")
+private val equalsRegex = Regex("^[^\\S\\r\\n]*=")
+
+/**
+ * Simple lightweight default options parser for external dependency resolvers.
+ *
+ * This parser expects the input to be a series of equality statements:
+ * `foo=Foo bar=Bar`
+ *
+ * And additionally supports flags without any equality statement:
+ * `foo bar`
+ */
+object EqualityExternalDependenciesResolverOptionsParser : ExternalDependenciesResolver.Options.Parser {
```

Comment:
"Equality" is too loaded here, I'd name it "Simple" instead.

Reply:
✅